test/message/processMIME.spec.ts

```diff
@@ -78,12 +79,11 @@ describe('processMIME', () => {
         expect(attachments.length).to.equal(1);
         const [attachment] = attachments;
         expect(attachment.fileName).to.equal('test.txt');
-        expect(attachment.generatedFileName).to.equal('test.txt');
         expect(attachment.contentType).to.equal('text/plain');
         expect(attachment.contentDisposition).to.equal('attachment');
-        expect(attachment.checksum).to.equal('94ee2b41f2016f2ec79a7b3a2faf920e');
+        expect(attachment.contentId.indexOf('pmcrypto')).to.not.equal(-1);
         expect(attachment.content).to.be.instanceOf(Uint8Array);
-        expect(utf8ArrayToString(attachment.content)).to.equal('this is the attachment text\r\n');
+        expect(utf8ArrayToString(attachment.content)).to.equal('this is the attachment text\n');
```

Comment:
Re: the change in EOL, apparently MailParser automatically made the conversion, but that was a bug, as I checked with their latest release and they also return only `\n` (which is the EOL originally used in the attachment).